pkg/tbtc/tbtc.go

```diff
@@ -146,16 +146,69 @@ func Initialize(
 				return
```

Comment:
One thing I realized when looking at the line earlier:
```
logger.Warnf(
					"DKG started event with seed [0x%x] and "+
						"starting block [%v] has been already processed",
					event.Seed,
					event.BlockNumber,
				)
```

I am not sure if this should be a `Warning`. The background event polling loop (see https://github.com/keep-network/keep-common/pull/63) will fetch this event multiple time, this is an expected behavior. I would consider changing the log level to `Info`.

Reply:
Also, the "starting block" part is confusing. I've got confused for a couple of minutes after reading this log message thinking we take a combo of seed+block number to duplicate events which would lead to duplicate keygen member processes being triggered.

Can we drop the `starting block [%v]` part?

This is a similar comment to [this one](https://github.com/keep-network/keep-core/pull/3427#discussion_r1061383154) made during the review of #3427.

Reply:
See 02888efc